game-core/src/main/java/org/triplea/debug/error/reporting/StackTraceErrorReportFormatter.java

```diff
@@ -158,11 +170,6 @@ private static String throwableToString(final Throwable e) {
     try (PrintWriter printWriter = new PrintWriter(outputStream, false, StandardCharsets.UTF_8)) {
       e.printStackTrace(printWriter);
     }
-    return "## Exception \n"
-        + e.getClass().getName()
-        + Optional.ofNullable(e.getMessage()).map(msg -> ": " + msg).orElse("")
-        + "\n\n## Stack Trace\n```\n"
-        + outputStream.toString(StandardCharsets.UTF_8)
-        + "\n```\n\n";
+    return "## Stack Trace\n```\n" + outputStream.toString(StandardCharsets.UTF_8) + "\n```\n\n";
```

Comment:
Something worth considering:
As we all know the report body is truncated at X characters. This means that the trailing `\n```\n\n` will be cut off if the stacktrace is too long.

Perhaps we want to truncate the stracktrace so we don't ever hit the other limit

Reply:
If you're missing the trailing ` ``` `, you'll still get the code format.

If we truncate only the stack trace, you'll need to calculate how much the rest of the content is taking and do a subtraction. 

It then seems just easier to truncate the whole thing knowing that the stack trace is at the end.

Reply:
True, just wanted to through the idea out there